sql/expression/function/time.go

```diff
@@ -694,6 +706,64 @@ func (n *Now) WithChildren(children ...sql.Expression) (sql.Expression, error) {
 	return NewNow(children...)
 }
 
+// UTCTimestamp is a function that returns the current time.
+type UTCTimestamp struct {
+	precision *int
+}
+
+// NewUTCTimestamp returns a new UTCTimestamp node.
+func NewUTCTimestamp(args ...sql.Expression) (sql.Expression, error) {
+	var precision *int
+	if len(args) > 1 {
+		return nil, sql.ErrInvalidArgumentNumber.New("UTC_TIMESTAMP", 1, len(args))
+	} else if len(args) == 1 {
+		precisionArg, err := sql.Int32.Convert(args[0])
```

Comment:
I apologize that I didn't catch this sooner. You need to call `args[0].Eval()` and convert the result of that, just like we're doing in `NewNow`.